src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Fund GHO Liquidity Committee"
+author: "TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-treasury-manage-gho-liquidity-committee/14914"
+---
+
+## Simple Summary
+
+This publication proposes the creation and funding of the GHO Liquidity Committee (GLC) for an initial 3-month period.
+
+## Motivation
+
+GHO is currently trading below its peg, with liquidity spread across various pools. To address this, the GLC will receive a budget of 5 ETH and 438,000 units of GHO with the primary objectives of:
```

Comment:
where does this number come from? proposal swaps 406k DAI to whatever currently is fair price - 50bps slippage, no?